daily-tags.sh

```diff
@@ -132,7 +132,7 @@ aliBuild --reference-sources mirror                    \
          --debug                                       \
          --work-dir "$workarea"                        \
          ${ARCHITECTURE:+--architecture $ARCHITECTURE} \
-         --jobs 8                                      \
+         --jobs 12                                     \
```

Comment:
This is fine, but don't we control that with a JENKINS variable as well? Maybe it should be $JOBS, no?